data/json/items/comestibles/junkfood.json

```diff
@@ -1290,5 +1290,21 @@
     "flags": [ "EATEN_HOT" ],
     "vitamins": [ [ "vitA", 3 ], [ "iron", 10 ] ],
     "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "chocolate_mre",
+    "name": { "str": "military chocolate bar" },
+    "copy_from": "chocolate",
+    "calories": 1800,
```

Comment:
This should be split over 3 bars.

> Three bars made up a daily ration and was intended to furnish the individual combat soldier with the 1,800 calories (7,500 kJ) minimum sustenance recommended each day.

>~The wikipedia article you used

Reply:
So 3 portions/charges.

Reply:
The values of "calories" and "weight" are those of a single charge, not three charges, so the caloric value here should be 600.

Additionally, a google search of "chocolate density" gives 0.641g/ml, so 115g * 3 charges of chocolate is approximately 500ml as opposed to 250ml.

Reply:
That's done. Original chocolate values are outside this PR's scope, possibly